src/work-list/work-list.component.tsx

```diff
@@ -146,6 +147,13 @@ const WorkList: React.FC<WorklistProps> = ({ fulfillerStatus }) => {
             </>
           ),
         },
+        patient: {
+          content: (
+            <>
+              <span>{entry.patient.display.split("-")[1]}</span>
```

Comment:
Was the split intended to get rid of the OpenMRS ID tagged to it?

Reply:
yes this was the intent